ads/aja.js

```diff
@@ -14,28 +14,18 @@
  * limitations under the License.
  */
 
-import {loadScript, validateData} from '../3p/3p';
+import {writeScript} from '../3p/3p';
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function aja(global, data) {
 
-  // ensure we have valid widgetIds value
-  validateData(data, ['widgetids']);
-
   (global._aja = global._aja || {
-    viewId: global.context.pageViewId,
-    widgetIds: data['widgetids'],
-    htmlURL: data['htmlurl'] || global.context.canonicalUrl,
-    ampURL: data['ampurl'] || global.context.sourceUrl,
-    fbk: data['fbk'] || '',
-    testMode: data['testmode'] || 'false',
-    styleFile: data['stylefile'] || '',
-    referrer: data['referrer'] || global.context.referrer,
+    sspCode: data['sspCode'],
```

Comment:
I've agreed on this https://github.com/ampproject/amphtml/pull/20652#issuecomment-465390283, but I want to bring this up one more time, just to be safe.  This is a breaking change for existing usage of `aja`, and we are good with it.

Reply:
really sorry. I do not seem to understand the meaning...

Would you like to open it again?
What should I do?

Reply:
That the existing usage of `<amp-embed type='aja'>` will be broken. Do you have customers that embed aja ad in AMP pages today? And how do you want to support them after the change.

Reply:
We have not yet delivered ads using <amp-embed type = 'aja'>....

I want to use <amp-ad> after this PR got through!

Reply:
Got it! Thank you for confirming.